### ISSUE TITLE
Missing Affy alternate CDFs

### ISSUE THREAD

**Issue by fieldima** [PavlidisLab/Gemma#92]
[Missing Affy alternates.txt](https://github.com/PavlidisLab/Gemma/files/4131311/Missing.Affy.alternates.txt)

Redirections of alternative platforms to parent platforms (affymetrix)

**Comment by ppavlidis** [comment]
We had most of these. I think there was one exception.

Then there is a mapping GPL6015 https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL6105 which is not an Affymetrix platform. For those cases, assuming the probe identifiers match across them, the proper action is to make a merged platform and then switch experiments to use it.

**Comment by fieldima** [comment]
Huh, that's odd that they were already mapped! The curation statuses must
not have been updated on Gemma when they were added the first time, oh
well, now we know. I'll check the other platforms listed as "Undergoing
citation" to see if there are any other instances of that.

**Comment by ppavlidis** [comment]
See https://github.com/PavlidisLab/Gemma/commit/83125462825f6521a40f6c8c133ef3d015a8c936

**Comment by fieldima** [comment]
Okay I went through and updated the GEMMA curation status of the platforms
I sent earlier today to reflect that they have already been updated to the
affy.altmappings.txt. I found one other instance of an already-listed
platform being marked as "Needs Attention". I also found one platform that
isn't listed on the master document. Its listed below but I can submit a
new issue as well.

GPL14996=GPL1261
